specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-10-27-preview/types/privateAccesses.json

```diff
@@ -0,0 +1,245 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PrivateAccesses types",
+    "version": "2023-10-27-preview"
+  },
+  "paths": {},
+  "definitions": {
+    "PrivateAccess": {
+      "type": "object",
+      "description": "PrivateAccesses tracked resource.",
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v2/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "systemData": {
```

Comment:
any reason why you are not using v5 of the common types? TrackedResource in v5 of common-types include systemdata already